pkg/httputils/httputils_test.go

```diff
@@ -37,7 +37,7 @@ func TestDownload400Errors(t *testing.T) {
 	defer ts.Close()
 	// Expected status code = 403
 	if _, err := Download(ts.URL); err == nil || err.Error() != expectedError {
-		t.Fatalf("Expected the the error %q, got %v", expectedError, err)
+		t.Fatalf("Expected the error %q, got %v", expectedError, err)
```

Comment:
just did a quick scan and I would say yes, and there was even a %s that could use it too on line 27.